<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GNMDEN3" failures="1" tests="2" name="Test" time="16.335" errors="0" timestamp="2023-05-04T22:29:32 IST">
  <testcase classname="TestCases.VerifyLogIN" name="InvalidLogin" time="0.792">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	error message is not valid expected [Please check your username.If you still can&amp;apos;t log in contact your Salesforce administrator.] but found [Please check your username and password. If you still can&amp;apos;t log in, contact your Salesforce administrator.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
error message is not valid expected [Please check your username.If you still can't log in contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]
at Resources.CommonMethods.handleAssertions(CommonMethods.java:19)
at TestCases.VerifyLogIN.InvalidLogin(VerifyLogIN.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- InvalidLogin -->
  <testcase classname="TestCases.VerifySignUp" name="SignUp" time="6.727"/>
</testsuite> <!-- Test -->
